src/test/kotlin/org/rust/ide/inspections/RsMainFunctionNotFoundInspectionTest.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections
+
+class RsMainFunctionNotFoundInspectionTest : RsInspectionsTestBase(RsMainFunctionNotFoundInspection::class) {
```

Comment:
Let's also add test that we don't highlight anything if file is not create root but it's in binary crate